CHANGELOG.md

```diff
@@ -1,8 +1,59 @@
+- Fix error when trying to clear non-existent component [#7215](https://github.com/rerun-io/rerun/pull/7215)
+- Correct dependency on `puffin` to 0.19.1, preventing a possible build failure [#7221](https://github.com/rerun-io/rerun/pull/7221)
+- Remove “has not yet been released” from migration-0-18.md [#7220](https://github.com/rerun-io/rerun/pull/7220)
+- Update `time` crate to 0.3.36, fixing compilation on newer Rust versions [#7228](https://github.com/rerun-io/rerun/pull/7228)
+- Typo fix - Rust - re_tracing [#7234](https://github.com/rerun-io/rerun/pull/7234)
+- Fix typo in arrow_cpp_install.md [#7235](https://github.com/rerun-io/rerun/pull/7235)
+- Dead code removal, renames, and clean-up [#7239](https://github.com/rerun-io/rerun/pull/7239)
+- Fix search index build [#7242](https://github.com/rerun-io/rerun/pull/7242)
+- Install sdk_info.h even if RERUN_INSTALL_RERUN_C option is OFF [#7246](https://github.com/rerun-io/rerun/pull/7246)
+- Combine `re_types` integration tests into one test target [#7250](https://github.com/rerun-io/rerun/pull/7250)
+- Convert most remaining `DataUI` component implementations to component uis in `re_component_ui` [#7247](https://github.com/rerun-io/rerun/pull/7247)
+- Fix references to closed tickets [#7219](https://github.com/rerun-io/rerun/pull/7219)
+- Fix gamma (srgb EOTF) for GLTF via `Asset3D` embedded rgb(a) textures [#7251](https://github.com/rerun-io/rerun/pull/7251)
+- Consistently call it "latest-at", with a dash [#7240](https://github.com/rerun-io/rerun/pull/7240)
+- Link to the per-language docs for `send_columns` [#7244](https://github.com/rerun-io/rerun/pull/7244)
+- Remove references to justfile [#7243](https://github.com/rerun-io/rerun/pull/7243)
+- Show single-line editor if there is no multiline editor [#7211](https://github.com/rerun-io/rerun/pull/7211)
+- Fix python version updating script [#7213](https://github.com/rerun-io/rerun/pull/7213)
+- Fix url to event.rs in `rerun analytics details` [#7230](https://github.com/rerun-io/rerun/pull/7230)
+- BGR(A) image format support [#7238](https://github.com/rerun-io/rerun/pull/7238)
+- Fix header levels in `sdk-operating-modes.md` [#7231](https://github.com/rerun-io/rerun/pull/7231)
+- Hide time controls if there is only one time point on a timeline [#7241](https://github.com/rerun-io/rerun/pull/7241)
+- Speed up data density graph by rendering them more coarsly [#7229](https://github.com/rerun-io/rerun/pull/7229)
+- Remove `Datatype` and `DatatypeBatch` [#7256](https://github.com/rerun-io/rerun/pull/7256)
+- Fix typo in spaces-and-transforms.md [#7255](https://github.com/rerun-io/rerun/pull/7255)
+- Remove `Loggable{Batch}::arrow_field` [#7257](https://github.com/rerun-io/rerun/pull/7257)
+- Remove `LoggableBatch::num_instances` [#7258](https://github.com/rerun-io/rerun/pull/7258)
+- re_arrow2 0.17.5 [#7262](https://github.com/rerun-io/rerun/pull/7262)
```

Comment:
This PR should have called out the user-facing change, i.e. the memory capacity leak